spaces/domains.json

```
@@ -123,5 +123,6 @@
   "gov.premia.finance": "premia.eth",
   "vote.perlinx.finance": "perlinx.eth",
   "vote.defigeek.xyz": "defigeek.eth",
-  "ambassador.synthetix.io": "snxambassador.eth"
+  "ambassador.synthetix.io": "snxambassador.eth",
+  "bamboodefi.com": "bamboodefi.eth"
```

Are you sure you want to point your Snapshot space to your main domain and not create a sub-domain instead? For example:
```suggestion
  "vote.bamboodefi.com": "bamboodefi.eth"
```

---

Yes, vote.bamboodefi.com is good.